kms_log/main.tf

```diff
@@ -10,11 +10,17 @@ data "aws_kms_key" "application"
     key_id = "alias/${var.env_name}-login-dot-gov-keymaker"
 }
 
+data "aws_sns_topic" "identity"
+{
+    name = "identity-events"
```

Comment:
Does TF allow us to read this from a variable?

Reply:
Yes, updated to variable.